src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -212,6 +215,10 @@ private static int doTypeSwitch(Object target, int startIndex, Object[] labels)
      * <p>
      * If no element in the {@code labels} array matches the target, then
      * the method of the call site return the length of the {@code labels} array.
+     * <p>
+     * The value of the {@code restart} index must be between {@code 0} and
```

Comment:
This description is a bit ambiguous - not clear whether it's `0 < x < length`, or `0 <= x < length`.